src/shared/modules/dbMeta/dbMetaEpics.ts

```diff
@@ -338,6 +368,35 @@ export const dbMetaEpic = (some$: any, store: any) =>
         .do(() => switchToRequestedDb(store))
         .mapTo({ type: DB_META_DONE })
     )
+export const dbCountEpic = (some$: any, store: any) =>
+  some$
+    .ofType(DB_META_DONE)
+    .filter(() => getCountAutomaticRefreshStatus(store.getState()))
```

Comment:
This could perhaps be called something like `getCondAutomaticRefreshEnabled`? (no need to change it)